src/PostgREST/OpenAPI.hs

```diff
@@ -183,9 +184,14 @@ makeRowFilters tn = map (makeRowFilter tn)
 makePathItem :: (Table, [Column], [Text]) -> (FilePath, PathItem)
 makePathItem (t, cs, _) = ("/" ++ unpack tn, p $ tableInsertable t)
   where
+    -- Use first line of table description as summary; rest as description (if present)
+    -- We strip leading newlines from description so that users can include a blank line between summary and description
+    (tSum, tDesc) = fmap fst &&& fmap (dropWhile (/='\n') . snd) $
+                    breakOn "\n" <$> tableDescription t
     tOp = (mempty :: Operation)
       & tags .~ Set.fromList [tn]
-      & description .~ tableDescription t
+      & summary .~ tSum
```

Comment:
Hmm, I'm confused. 😕

> if no comment is set, emit no summary

I don't know whether this is true, but I was assuming that the way to do this would be to make the summary a Nothing value, which would cause it to be omitted in the OpenAPI output. Whereas a present value of the empty string for summary would it to be present in the OpenAPI output but empty.

How about adding test cases just to verify that this is causing the intended behavior?